articles/quickstart/native/android/00-centralized-login.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Centralized Login
+description: This tutorial will show you how to integrate the Auth0 Centralized Login in your Android project in order to present the login box.
+budicon: 448
+---
+
+This tutorial will show you how to integrate the Auth0 Centralized Login in your Android project in order to present the login box.
+
+<%= include('../../../_includes/_package', {
+  org: 'auth0-samples',
+  repo: 'auth0-android-sample',
+  path: '00-centralized-login',
+  requirements: [
+    'Android Studio 2.3',
+    'Android SDK 25',
+    'Emulator - Nexus 5X - Android 6.0'
+  ]
+}) %>
+
+<%= include('_includes/_auth0') %>
+__
+
+
+### Start the authentication
```

Comment:
Use level 2 headings and capitalize bigger words:
`## Start the Authentication`